pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -115,8 +127,12 @@ type Registry struct {
 	Blocked bool `toml:"blocked,omitempty"`
 	// If true, mirrors will only be used for digest pulls. Pulling images by
 	// tag can potentially yield different images, depending on which endpoint
-	// we pull from.  Forcing digest-pulls for mirrors avoids that issue.
+	// we pull from.  Restricting mirrors to pulls for mirrors avoids that issue.
```

Comment:
I don’t understand what “Restricting mirrors to pulls for mirrors” means. The previous version was not quite correct either, to be fair — but it should be possible to do better than both.

(See also elsewhere about making the rationale / recommendations use-visible.)

Reply:
My mistake. It should be `Restricting mirrors to pulls by digest mirrors avoids that issue.`

Reply:
This is still outstanding.